framework/auth/oauth_scopes.py

```diff
@@ -124,7 +124,7 @@ class ComposedScopes(object):
     USERS_READ = (CoreScopes.USERS_READ, CoreScopes.ALWAYS_PUBLIC, )
     USERS_WRITE = USERS_READ + (CoreScopes.USERS_WRITE,)
     USERS_CREATE = USERS_READ + (CoreScopes.USERS_CREATE, )
-    USERS_EMAIL = (CoreScopes.USERS_EMAIL, )
+    USER_EMAIL_READ =  (CoreScopes.USERS_READ, CoreScopes.USER_EMAIL_READ, )
```

Comment:
I'm fine with reverting this, but couldn't [`description`](https://github.com/caseyrollins/osf.io/blob/8577473cda024defad5fb37ae52abfede05a36b4/framework/auth/oauth_scopes.py#L214) just be updated to be more explicit (i.e. "Read your profile data and primary email address")?